### ISSUE TITLE
 [Android][Fabric] Debug build crashes with "RNSScreenStackHeaderConfig" was not found in the UIManager

### ISSUE THREAD

**Issue by BritCoder96** [software-mansion/react-native-screens#1712]
### Description

Running a fresh install of React Navigation 6, on React Native 0.71.1, with the new architecture and hermes enabled. Getting the error in the title on app open in debug mode. I've read every post I can find on GitHub and StackOverflow on this (including a number of similar issues here), but nothing is working. From what I'm reading, this error usually implies that I don't have all the required libraries installed, but I do (react-native-screens, react-native-reanimated, react-native-gesture-handler, react-native-pager-view, react-native-safe-area-context, and react-native-masked-view), all at their latest versions, but still getting this error even after clearing the metro cache, the gradle cache, and the npm cache (including deleting and re-installing node_modules). Kind of stumped at this point, and I have a deadline I'm trying to meet for this project. Any help would be appreciated. Thanks!!!

The relevant part of my package.json is below. Feel free to ask for any additional code and I'll be happy to add it. My project is in a private repo for legal reasons so I can't share the whole thing, but I can share anonymized generic code if need be.

``` 
  "scripts": {
    "start": "node node_modules/react-native/local-cli/cli.js start",
    "test": "jest"
  },
  "dependencies": {
    "@react-native-firebase/analytics": "^17.0.0",
    "@react-native-firebase/app": "^17.0.0",
    "@react-native-firebase/crashlytics": "^17.0.0",
    "@react-native-firebase/messaging": "^17.0.0",
    "@react-native-masked-view/masked-view": "^0.2.8",
    "@react-navigation/bottom-tabs": "^6.5.4",
    "@react-navigation/material-top-tabs": "^6.5.3",
    "@react-navigation/native": "^6.1.3",
    "@react-navigation/native-stack": "^6.9.9",
    "@react-navigation/stack": "^6.3.12",
    "babel": "^6.23.0",
    "debounce": "^1.1.0",
    "install": "^0.13.0",
    "jetify": "^1.0.2",
    "moment": "^2.19.1",
    "npm": "^9.3.1",
    "react": "18.2.0",
    "react-moment": "^1.1.3",
    "react-native": "0.71.1",
    "react-native-datepicker": "^1.4.5",
    "react-native-dropdown": "0.0.6",
    "react-native-extended-stylesheet": "^0.3.2",
    "react-native-gesture-handler": "^2.9.0",
    "react-native-modal-datetime-picker": "^14.0.1",
    "react-native-modal-selector": "2.1.2",
    "react-native-pager-view": "^6.1.2",
    "react-native-reanimated": "^3.0.0-rc.3",
     "react-native-responsive-image": "^2.1.0",
    "react-native-safe-area-context": "^4.5.0",
    "react-native-screens": "^3.19.0",
    "react-native-search-bar": "^3.0.0",
    "react-native-select-multiple": "^1.0.4",
    "react-native-sound": "^0.10.3",
    "react-native-tab-view": "^3.3.4",
    "react-native-vector-icons": "^9.2.0",
    "react-timer-mixin": "^0.13.3",
    "xmlbuilder": "^9.0.1"
  },
,
  "devDependencies": {
    "babel-jest": "^29.4.1",
    "jest": "^29.3.1",
    "jetifier": "^2.0.0",
    "metro": "^0.75.0",
    "metro-react-native-babel-preset": "0.75.0"
  },
  "jest": {
    "preset": "react-native"
  },
  "rnpm": {
    "assets": [
      "fonts"
    ]
  },
  "postinstall": "jetify"
}
```

### Steps to reproduce

Fresh install of React Native 0.71.1, with react-navigation (and the rest of the dependencies) as listed in the package.json above. 

Basic initial home screen:

```
import React from 'react';

import 'react-native-gesture-handler';

import Homepage from './Homepage';

import { NavigationContainer } from '@react-navigation/native';
import { createNativeStackNavigator } from '@react-navigation/native-stack';

export default function App() {

  const Stack = createNativeStackNavigator();

  return (
    <NavigationContainer>
      <Stack.Navigator initialRouteName="Home">
        <Stack.Screen name="Home" component={Homepage} />
      </Stack.Navigator>
    </NavigationContainer>
  )
}
```

### Snack or a link to a repository
https://github.com/BritCoder96/sample_repro

### Screens version

4.5.0

### React Native version

0.71.1

### Platforms

Android

### JavaScript runtime

Hermes

### Workflow

React Native (without Expo)

### Architecture

Fabric (New Architecture)

### Build type

Debug mode

### Device

Real device

### Device model

Samsung Galaxy S20 Android 13

### Acknowledgements

Yes

**Comment by kkafar** [comment]
Hi @BritCoder96, 

please run `yarn react-native config` in your app main directory provide me with output of that command! Thanks

**Comment by BritCoder96** [comment]
> Hi @BritCoder96,
> 
> please run `yarn react-native config` in your app main directory and provide me with output of that command! Thanks

@kkafar Thanks! Please see the output below. NOTE: I've replaced the name of my package in this output with 'test' to protect the name of my product, which I would prefer to keep under wraps for now. Let me know if you need anything else. Thank you!

```
$ yarn react-native config
yarn run v1.22.19
$ /home/bryan/test/Test/node_modules/.bin/react-native config
{
  "root": "/home/bryan/test/Test",
  "reactNativePath": "/home/bryan/test/Test/node_modules/react-native",
  "dependencies": {
    "@react-native-firebase/analytics": {
      "root": "/home/bryan/test/Test/node_modules/@react-native-firebase/analytics",
      "name": "@react-native-firebase/analytics",
      "platforms": {
        "ios": {
          "podspecPath": "/home/bryan/test/Test/node_modules/@react-native-firebase/analytics/RNFBAnalytics.podspec",
          "configurations": [],
          "scriptPhases": []
        },
        "android": {
          "sourceDir": "/home/bryan/test/Test/node_modules/@react-native-firebase/analytics/android",
          "packageImportPath": "import io.invertase.firebase.analytics.ReactNativeFirebaseAnalyticsPackage;",
          "packageInstance": "new ReactNativeFirebaseAnalyticsPackage()",
          "buildTypes": [],
          "componentDescriptors": [],
          "androidMkPath": "/home/bryan/test/Test/node_modules/@react-native-firebase/analytics/android/build/generated/source/codegen/jni/Android.mk",
          "cmakeListsPath": "/home/bryan/test/Test/node_modules/@react-native-firebase/analytics/android/build/generated/source/codegen/jni/CMakeLists.txt"
        }
      }
    },
    "@react-native-firebase/app": {
      "root": "/home/bryan/test/Test/node_modules/@react-native-firebase/app",
      "name": "@react-native-firebase/app",
      "platforms": {
        "ios": {
          "podspecPath": "/home/bryan/test/Test/node_modules/@react-native-firebase/app/RNFBApp.podspec",
          "configurations": [],
          "scriptPhases": [
            {
              "name": "[RNFB] Core Configuration",
              "path": "./ios_config.sh",
              "execution_position": "after_compile",
              "input_files": [
                "$(BUILT_PRODUCTS_DIR)/$(INFOPLIST_PATH)"
              ]
            }
          ]
        },
        "android": {
          "sourceDir": "/home/bryan/test/Test/node_modules/@react-native-firebase/app/android",
          "packageImportPath": "import io.invertase.firebase.app.ReactNativeFirebaseAppPackage;",
          "packageInstance": "new ReactNativeFirebaseAppPackage()",
          "buildTypes": [],
          "componentDescriptors": [],
          "androidMkPath": "/home/bryan/test/Test/node_modules/@react-native-firebase/app/android/build/generated/source/codegen/jni/Android.mk",
          "cmakeListsPath": "/home/bryan/test/Test/node_modules/@react-native-firebase/app/android/build/generated/source/codegen/jni/CMakeLists.txt"
        }
      }
    },
    "@react-native-firebase/crashlytics": {
      "root": "/home/bryan/test/Test/node_modules/@react-native-firebase/crashlytics",
      "name": "@react-native-firebase/crashlytics",
      "platforms": {
        "ios": {
          "podspecPath": "/home/bryan/test/Test/node_modules/@react-native-firebase/crashlytics/RNFBCrashlytics.podspec",
          "configurations": [],
          "scriptPhases": [
            {
              "name": "[RNFB] Crashlytics Configuration",
              "path": "./ios_config.sh",
              "execution_position": "after_compile",
              "input_files": [
                "${DWARF_DSYM_FOLDER_PATH}/${DWARF_DSYM_FILE_NAME}/Contents/Resources/DWARF/${TARGET_NAME}",
                "$(SRCROOT)/$(BUILT_PRODUCTS_DIR)/$(INFOPLIST_PATH)"
              ]
            }
          ]
        },
        "android": {
          "sourceDir": "/home/bryan/test/Test/node_modules/@react-native-firebase/crashlytics/android",
          "packageImportPath": "import io.invertase.firebase.crashlytics.ReactNativeFirebaseCrashlyticsPackage;",
          "packageInstance": "new ReactNativeFirebaseCrashlyticsPackage()",
          "buildTypes": [],
          "componentDescriptors": [],
          "androidMkPath": "/home/bryan/test/Test/node_modules/@react-native-firebase/crashlytics/android/build/generated/source/codegen/jni/Android.mk",
          "cmakeListsPath": "/home/bryan/test/Test/node_modules/@react-native-firebase/crashlytics/android/build/generated/source/codegen/jni/CMakeLists.txt"
        }
      }
    },
    "@react-native-firebase/messaging": {
      "root": "/home/bryan/test/Test/node_modules/@react-native-firebase/messaging",
      "name": "@react-native-firebase/messaging",
      "platforms": {
        "ios": {
          "podspecPath": "/home/bryan/test/Test/node_modules/@react-native-firebase/messaging/RNFBMessaging.podspec",
          "configurations": [],
          "scriptPhases": []
        },
        "android": {
          "sourceDir": "/home/bryan/test/Test/node_modules/@react-native-firebase/messaging/android",
          "packageImportPath": "import io.invertase.firebase.messaging.ReactNativeFirebaseMessagingPackage;",
          "packageInstance": "new ReactNativeFirebaseMessagingPackage()",
          "buildTypes": [],
          "componentDescriptors": [],
          "androidMkPath": "/home/bryan/test/Test/node_modules/@react-native-firebase/messaging/android/build/generated/source/codegen/jni/Android.mk",
          "cmakeListsPath": "/home/bryan/test/Test/node_modules/@react-native-firebase/messaging/android/build/generated/source/codegen/jni/CMakeLists.txt"
        }
      }
    },
    "@react-native-masked-view/masked-view": {
      "root": "/home/bryan/test/Test/node_modules/@react-native-masked-view/masked-view",
      "name": "@react-native-masked-view/masked-view",
      "platforms": {
        "ios": {
          "podspecPath": "/home/bryan/test/Test/node_modules/@react-native-masked-view/masked-view/RNCMaskedView.podspec",
          "configurations": [],
          "scriptPhases": []
        },
        "android": {
          "sourceDir": "/home/bryan/test/Test/node_modules/@react-native-masked-view/masked-view/android",
          "packageImportPath": "import org.reactnative.maskedview.RNCMaskedViewPackage;",
          "packageInstance": "new RNCMaskedViewPackage()",
          "buildTypes": [],
          "componentDescriptors": [],
          "androidMkPath": "/home/bryan/test/Test/node_modules/@react-native-masked-view/masked-view/android/build/generated/source/codegen/jni/Android.mk",
          "cmakeListsPath": "/home/bryan/test/Test/node_modules/@react-native-masked-view/masked-view/android/build/generated/source/codegen/jni/CMakeLists.txt"
        }
      }
    },
    "react-native-gesture-handler": {
      "root": "/home/bryan/test/Test/node_modules/react-native-gesture-handler",
      "name": "react-native-gesture-handler",
      "platforms": {
        "ios": {
          "podspecPath": "/home/bryan/test/Test/node_modules/react-native-gesture-handler/RNGestureHandler.podspec",
          "configurations": [],
          "scriptPhases": []
        },
        "android": {
          "sourceDir": "/home/bryan/test/Test/node_modules/react-native-gesture-handler/android",
          "packageImportPath": "import com.swmansion.gesturehandler.RNGestureHandlerPackage;",
          "packageInstance": "new RNGestureHandlerPackage()",
          "buildTypes": [],
          "libraryName": "rngesturehandler_codegen",
          "componentDescriptors": [
            "RNGestureHandlerButtonComponentDescriptor",
            "RNGestureHandlerRootViewComponentDescriptor"
          ],
          "androidMkPath": "/home/bryan/test/Test/node_modules/react-native-gesture-handler/android/build/generated/source/codegen/jni/Android.mk",
          "cmakeListsPath": "/home/bryan/test/Test/node_modules/react-native-gesture-handler/android/build/generated/source/codegen/jni/CMakeLists.txt"
        }
      }
    },
    "react-native-pager-view": {
      "root": "/home/bryan/test/Test/node_modules/react-native-pager-view",
      "name": "react-native-pager-view",
      "platforms": {
        "ios": {
          "podspecPath": "/home/bryan/test/Test/node_modules/react-native-pager-view/react-native-pager-view.podspec",
          "configurations": [],
          "scriptPhases": []
        },
        "android": {
          "sourceDir": "/home/bryan/test/Test/node_modules/react-native-pager-view/android",
          "packageImportPath": "import com.reactnativepagerview.PagerViewPackage;",
          "packageInstance": "new PagerViewPackage()",
          "buildTypes": [],
          "libraryName": "RNCViewPager",
          "componentDescriptors": [
            "RNCViewPagerComponentDescriptor"
          ],
          "androidMkPath": "/home/bryan/test/Test/node_modules/react-native-pager-view/android/build/generated/source/codegen/jni/Android.mk",
          "cmakeListsPath": "/home/bryan/test/Test/node_modules/react-native-pager-view/android/build/generated/source/codegen/jni/CMakeLists.txt"
        }
      }
    },
    "react-native-reanimated": {
      "root": "/home/bryan/test/Test/node_modules/react-native-reanimated",
      "name": "react-native-reanimated",
      "platforms": {
        "ios": {
          "podspecPath": "/home/bryan/test/Test/node_modules/react-native-reanimated/RNReanimated.podspec",
          "configurations": [],
          "scriptPhases": []
        },
        "android": {
          "sourceDir": "/home/bryan/test/Test/node_modules/react-native-reanimated/android",
          "packageImportPath": "import com.swmansion.reanimated.ReanimatedPackage;",
          "packageInstance": "new ReanimatedPackage()",
          "buildTypes": [],
          "componentDescriptors": [],
          "androidMkPath": "/home/bryan/test/Test/node_modules/react-native-reanimated/android/build/generated/source/codegen/jni/Android.mk",
          "cmakeListsPath": "/home/bryan/test/Test/node_modules/react-native-reanimated/android/build/generated/source/codegen/jni/CMakeLists.txt"
        }
      }
    },
    "react-native-safe-area-context": {
      "root": "/home/bryan/test/Test/node_modules/react-native-safe-area-context",
      "name": "react-native-safe-area-context",
      "platforms": {
        "ios": {
          "podspecPath": "/home/bryan/test/Test/node_modules/react-native-safe-area-context/react-native-safe-area-context.podspec",
          "configurations": [],
          "scriptPhases": []
        },
        "android": {
          "sourceDir": "/home/bryan/test/Test/node_modules/react-native-safe-area-context/android",
          "packageImportPath": "import com.th3rdwave.safeareacontext.SafeAreaContextPackage;",
          "packageInstance": "new SafeAreaContextPackage()",
          "buildTypes": [],
          "libraryName": "safeareacontext",
          "componentDescriptors": [
            "RNCSafeAreaProviderComponentDescriptor",
            "RNCSafeAreaViewComponentDescriptor"
          ],
          "androidMkPath": "/home/bryan/test/Test/node_modules/react-native-safe-area-context/android/build/generated/source/codegen/jni/Android.mk",
          "cmakeListsPath": "/home/bryan/test/Test/node_modules/react-native-safe-area-context/android/src/main/jni/CMakeLists.txt"
        }
      }
    },
    "react-native-screens": {
      "root": "/home/bryan/test/Test/node_modules/react-native-screens",
      "name": "react-native-screens",
      "platforms": {
        "ios": {
          "podspecPath": "/home/bryan/test/Test/node_modules/react-native-screens/RNScreens.podspec",
          "configurations": [],
          "scriptPhases": []
        },
        "android": {
          "sourceDir": "/home/bryan/test/Test/node_modules/react-native-screens/android",
          "packageImportPath": "import com.swmansion.rnscreens.RNScreensPackage;",
          "packageInstance": "new RNScreensPackage()",
          "buildTypes": [],
          "libraryName": "rnscreens",
          "componentDescriptors": [
            "RNSFullWindowOverlayComponentDescriptor",
            "RNSScreenContainerComponentDescriptor",
            "RNSScreenNavigationContainerComponentDescriptor",
            "RNSScreenStackHeaderConfigComponentDescriptor",
            "RNSScreenStackHeaderSubviewComponentDescriptor",
            "RNSScreenStackComponentDescriptor",
            "RNSSearchBarComponentDescriptor",
            "RNSScreenComponentDescriptor"
          ],
          "androidMkPath": "/home/bryan/test/Test/node_modules/react-native-screens/android/build/generated/source/codegen/jni/Android.mk",
          "cmakeListsPath": "/home/bryan/test/Test/node_modules/react-native-screens/android/src/main/jni/CMakeLists.txt"
        }
      }
    },
    "react-native-search-bar": {
      "root": "/home/bryan/test/Test/node_modules/react-native-search-bar",
      "name": "react-native-search-bar",
      "platforms": {
        "ios": {
          "podspecPath": "/home/bryan/test/Test/node_modules/react-native-search-bar/RNSearchBar.podspec",
          "configurations": [],
          "scriptPhases": []
        },
        "android": {
          "sourceDir": "/home/bryan/test/Test/node_modules/react-native-search-bar/android",
          "packageImportPath": "import org.umhan35.RNSearchBarPackage;",
          "packageInstance": "new RNSearchBarPackage()",
          "buildTypes": [],
          "componentDescriptors": [],
          "androidMkPath": "/home/bryan/test/Test/node_modules/react-native-search-bar/android/build/generated/source/codegen/jni/Android.mk",
          "cmakeListsPath": "/home/bryan/test/Test/node_modules/react-native-search-bar/android/build/generated/source/codegen/jni/CMakeLists.txt"
        }
      }
    },
    "react-native-sound": {
      "root": "/home/bryan/test/Test/node_modules/react-native-sound",
      "name": "react-native-sound",
      "platforms": {
        "ios": {
          "podspecPath": "/home/bryan/test/Test/node_modules/react-native-sound/RNSound.podspec",
          "configurations": [],
          "scriptPhases": []
        },
        "android": {
          "sourceDir": "/home/bryan/test/Test/node_modules/react-native-sound/android",
          "packageImportPath": "import com.zmxv.RNSound.RNSoundPackage;",
          "packageInstance": "new RNSoundPackage()",
          "buildTypes": [],
          "componentDescriptors": [],
          "androidMkPath": "/home/bryan/test/Test/node_modules/react-native-sound/android/build/generated/source/codegen/jni/Android.mk",
          "cmakeListsPath": "/home/bryan/test/Test/node_modules/react-native-sound/android/build/generated/source/codegen/jni/CMakeLists.txt"
        }
      }
    },
    "react-native-vector-icons": {
      "root": "/home/bryan/test/Test/node_modules/react-native-vector-icons",
      "name": "react-native-vector-icons",
      "platforms": {
        "ios": {
          "podspecPath": "/home/bryan/test/Test/node_modules/react-native-vector-icons/RNVectorIcons.podspec",
          "configurations": [],
          "scriptPhases": []
        },
        "android": {
          "sourceDir": "/home/bryan/test/Test/node_modules/react-native-vector-icons/android",
          "packageImportPath": "import com.oblador.vectoricons.VectorIconsPackage;",
          "packageInstance": "new VectorIconsPackage()",
          "buildTypes": [],
          "componentDescriptors": [],
          "androidMkPath": "/home/bryan/test/Test/node_modules/react-native-vector-icons/android/build/generated/source/codegen/jni/Android.mk",
          "cmakeListsPath": "/home/bryan/test/Test/node_modules/react-native-vector-icons/android/build/generated/source/codegen/jni/CMakeLists.txt"
        }
      }
    }
  },
  "commands": [
    {
      "name": "log-ios",
      "description": "starts iOS device syslog tail"
    },
    {
      "name": "run-ios",
      "description": "builds your app and starts it on iOS simulator",
      "examples": [
        {
          "desc": "Run on a different simulator, e.g. iPhone SE (2nd generation)",
          "cmd": "react-native run-ios --simulator \"iPhone SE (2nd generation)\""
        },
        {
          "desc": "Run on a connected device, e.g. Max's iPhone",
          "cmd": "react-native run-ios --device \"Max's iPhone\""
        },
        {
          "desc": "Run on the AppleTV simulator",
          "cmd": "react-native run-ios --simulator \"Apple TV\"  --scheme \"helloworld-tvOS\""
        }
      ],
      "options": [
        {
          "name": "--simulator <string>",
          "description": "Explicitly set simulator to use. Optionally include iOS version between parenthesis at the end to match an exact version: \"iPhone 6 (10.0)\"",
          "default": "iPhone 14"
        },
        {
          "name": "--configuration <string>",
          "description": "Explicitly set the scheme configuration to use",
          "default": "Debug"
        },
        {
          "name": "--scheme <string>",
          "description": "Explicitly set Xcode scheme to use"
        },
        {
          "name": "--device [string]",
          "description": "Explicitly set device to use by name.  The value is not required if you have a single device connected."
        },
        {
          "name": "--udid <string>",
          "description": "Explicitly set device to use by udid"
        },
        {
          "name": "--no-packager",
          "description": "Do not launch packager while building"
        },
        {
          "name": "--verbose",
          "description": "Do not use xcbeautify or xcpretty even if installed"
        },
        {
          "name": "--port <number>",
          "default": 8081
        },
        {
          "name": "--binary-path <string>",
          "description": "Path relative to project root where pre-built .app binary lives."
        },
        {
          "name": "--terminal <string>",
          "description": "Launches the Metro Bundler in a new window using the specified terminal path."
        },
        {
          "name": "--xcconfig [string]",
          "description": "Explicitly set xcconfig to use"
        },
        {
          "name": "--buildFolder <string>",
          "description": "Location for iOS build artifacts. Corresponds to Xcode's \"-derivedDataPath\"."
        }
      ]
    },
    {
      "name": "log-android",
      "description": "starts logkitty"
    },
    {
      "name": "run-android",
      "description": "builds your app and starts it on a connected Android emulator or device",
      "options": [
        {
          "name": "--mode <string>",
          "description": "Specify your app's build variant"
        },
        {
          "name": "--variant <string>",
          "description": "Specify your app's build variant. Deprecated! Use 'mode' instead"
        },
        {
          "name": "--no-packager",
          "description": "Do not launch packager while building"
        },
        {
          "name": "--port <number>",
          "default": 8081
        },
        {
          "name": "--terminal <string>",
          "description": "Launches the Metro Bundler in a new window using the specified terminal path.",
          "default": "xterm-256color"
        },
        {
          "name": "--tasks <list>",
          "description": "Run custom Gradle tasks. By default it's \"assembleDebug\". Will override passed mode and variant arguments."
        },
        {
          "name": "--active-arch-only",
          "description": "Build native libraries only for the current device architecture for debug builds.",
          "default": false
        },
        {
          "name": "--extra-params <string>",
          "description": "Custom properties passed to gradle build command"
        },
        {
          "name": "--appId <string>",
          "description": "Specify an applicationId to launch after build. If not specified, `package` from AndroidManifest.xml will be used.",
          "default": ""
        },
        {
          "name": "--appIdSuffix <string>",
          "description": "Specify an applicationIdSuffix to launch after build.",
          "default": ""
        },
        {
          "name": "--main-activity <string>",
          "description": "Name of the activity to start",
          "default": "MainActivity"
        },
        {
          "name": "--deviceId <string>",
          "description": "builds your app and starts it on a specific device/simulator with the given device id (listed by running \"adb devices\" on the command line)."
        },
        {
          "name": "--list-devices",
          "description": "Lists all available Android devices and simulators and let you choose one to run the app",
          "default": false
        },
        {
          "name": "--binary-path <string>",
          "description": "Path relative to project root where pre-built .apk binary lives."
        }
      ]
    },
    {
      "name": "build-android",
      "description": "builds your app",
      "options": [
        {
          "name": "--mode <string>",
          "description": "Specify your app's build variant"
        },
        {
          "name": "--variant <string>",
          "description": "Specify your app's build variant. Deprecated! Use 'mode' instead"
        },
        {
          "name": "--no-packager",
          "description": "Do not launch packager while building"
        },
        {
          "name": "--port <number>",
          "default": 8081
        },
        {
          "name": "--terminal <string>",
          "description": "Launches the Metro Bundler in a new window using the specified terminal path.",
          "default": "xterm-256color"
        },
        {
          "name": "--tasks <list>",
          "description": "Run custom Gradle tasks. By default it's \"assembleDebug\". Will override passed mode and variant arguments."
        },
        {
          "name": "--active-arch-only",
          "description": "Build native libraries only for the current device architecture for debug builds.",
          "default": false
        },
        {
          "name": "--extra-params <string>",
          "description": "Custom properties passed to gradle build command"
        }
      ]
    }
  ],
  "healthChecks": [],
  "platforms": {
    "ios": {},
    "android": {}
  },
  "project": {
    "ios": null,
    "android": {
      "sourceDir": "/home/bryan/test/Test/android",
      "appName": "app",
      "packageName": "com.Test.Test"
    }
  }
}
Done in 8.02s.
```

**Comment by kkafar** [comment]
@BritCoder96

I've tested it and everything seems to be working fine. 

This issue indicates that some native components (`RNSScreenStackHeaderConfig`) have not been registered in `UIManager` -> there was most likely some issue with library linking / RNCLI not detecting our native components. But seeing the result of `yarn react-native config` it seems that the RNCLI properly sees our components. 

Do you think it would be possible for you to create some separate repo with issue reproduction? 
As for now, w/o proper reproduction I don't think I'll be able to do much about the issue.

**Comment by BritCoder96** [comment]
> @kkafar I tried with react-native-screens 3.18.2, same error. yarn global list / npm -g list doesn't return anything.

I made the sample repo here: https://github.com/BritCoder96/sample_repro, though I'm not 100% sure it's exactly the same issue there, the config is pretty much the same, so if there's an issue you see in there that might be it let me know. Thanks!

**Comment by BritCoder96** [comment]
@kkafar Any update on this?

**Comment by p13i** [comment]
I am facing the same bug but with an Expo-managed build.

**Comment by kkafar** [comment]
@BritCoder96,
I've spent some time trying to get this repro working but I failed.

Few tips: 

1. You can't name the app "test" (see `package.json`) - this name conflicts with some node core modules.
2. [Merging artifacts](https://github.com/BritCoder96/sample_repro/blob/14740eee73552e45c3ed8eacb2eca1ee81334498/android/app/BUCK#L47-L65) do not help.
3. Your `build.gradle` files do not comply to ReactNative's templates for `0.71` version. If you're updating your React Native version there are usually few more steps to do than only bumping version number in `package.json`. [Upgrade helper tool](https://react-native-community.github.io/upgrade-helper/?from=0.69.6&to=0.71.2) might be of some help (please select appropriate RN versions).
4. Following on 3.: It is not only down to `build.gradle` files, it would be reasonable to update whole project.
5. It is undesired to specify library dependencies `:react-native-xxx` [here]
(https://github.com/BritCoder96/sample_repro/blob/14740eee73552e45c3ed8eacb2eca1ee81334498/android/app/build.gradle#L139-L169) manually. You should let the autolinking mechanism take care of this (it may be case of some problems).

IMO updating your project to structure appropriate to given RN version should resolve the issue, but in case further help is needed please provide me with **working** issue reproduction :pray:

**Comment by kkafar** [comment]
@p13i 
Would you mind providing a reproduction?

**Comment by BritCoder96** [comment]
> @BritCoder96, I've spent some time trying to get this repro working but I failed.
> 
> Few tips:
> 
> 1. You can't name the app "test" (see `package.json`) - this name conflicts with some node core modules.
> 2. [Merging artifacts](https://github.com/BritCoder96/sample_repro/blob/14740eee73552e45c3ed8eacb2eca1ee81334498/android/app/BUCK#L47-L65) do not help.
> 3. Your `build.gradle` files do not comply to ReactNative's templates for `0.71` version. If you're updating your React Native version there are usually few more steps to do than only bumping version number in `package.json`. [Upgrade helper tool](https://react-native-community.github.io/upgrade-helper/?from=0.69.6&to=0.71.2) might be of some help (please select appropriate RN versions).
> 4. Following on 3.: It is not only down to `build.gradle` files, it would be reasonable to update whole project.
> 5. It is undesired to specify library dependencies `:react-native-xxx` [here]
>    (https://github.com/BritCoder96/sample_repro/blob/14740eee73552e45c3ed8eacb2eca1ee81334498/android/app/build.gradle#L139-L169) manually. You should let the autolinking mechanism take care of this (it may be case of some problems).
> 
> IMO updating your project to structure appropriate to given RN version should resolve the issue, but in case further help is needed please provide me with **working** issue reproduction 🙏

@kkafar With the help of your links on upgrading react native, I've got this working. Thank you for all you help!!!

**Comment by p13i** [comment]
Hi @kkafar , I finally figured out that in my case (for an Expo-managed app, not bare workflow) I was missing the `react-native-screens` dependency in my `package.json` file. I removed it as part of a clean up effort to reduce the number of extraneous dependencies I used but I had no idea that this explicit dependency was required for Expo to add `RNScreens` to the generated Podfile install.

TLDR: My solution isn't relevant here but may help someone else.

**Comment by kkafar** [comment]
> With the help of your links on upgrading react native, I've got this working. Thank you for all you help!!!

Great to hear that!

> I finally figured out that in my case (for an Expo-managed app, not bare workflow) I was missing the react-native-screens dependency in my package.json file. I removed it as part of a clean up effort to reduce the number of extraneous dependencies I used but I had no idea that this explicit dependency was required for Expo to add RNScreens to the generated Podfile install.

Same here!

Closing the issue

**Comment by allemanfredi** [comment]
same here

**Comment by Gustavo-Marques19** [comment]
same here
